### PR TITLE
feat: Update to scrape favicon

### LINKED ISSUE #15: Failed to fetch images that accessed in background-image
**Describe the bug**

**To Reproduce**
1. Go to https://pycontw.github.io/pycon_archive_past_website/2020/zh-hant/ (same as 2016-2019)
2. See the errors that showed in console like image below
image

**Expected behavior**
Successfully access all background image that with customized base_url

### PR DESCRIPTION
**Pull request by josix**
- [x] Update to download favicon of each year site

Closes #15

### CODE REVIEW

**Review by allen91wu — CHANGES_REQUESTED** [review]
Hi @josix,
I have leave a comment for your reference. Please feel free to tell me any concern.

File: main.py [hunk 1]
```diff
@@ -88,6 +88,23 @@ def img(soup):
                 mkdir(person['photo'])
                 writefile(person['photo'])
 
+def get_assets(path: Path):
+    """
+    Get all assets in the given path.
+    """
+    export_path: Path = Path.cwd().joinpath(*path.parts)
```
[the remark this answers is not in the document]
Comment (josix) [reply main.py:95]: I kept this line for the unexpected url path, maybe I could add a validation and ignore the invalid input.

[the remark this answers is not in the document]
Comment (josix) [reply main.py:95]: Yeah, thanks for yhe advice.

**Comment by josix** [comment]
Hello @allen91wu I've updated it following the suggestion, please have a look. Thank you.

**Review by allen91wu — APPROVED** [review]
LGTM!

**Pull request merged**